### DISCUSSION TITLE
How to setup sdrpp for MSI2500 support?

### DISCUSSION THREAD

**Discussion by dl5raz** [AlexandreRouma/SDRPlusPlus#1459] [Q&A]
my hardware: lsusb output
Bus 001 Device 016: ID 1df7:2500 SDRplay RSP1
How to setup on Linux?
Any help are welcome

**Comment by KubaPro010** [comment]
download sdrplay api (and then use the sdrplay source module in sdrpp)

**Reply by dl5raz** [reply]
The Hardware: Bus 001 Device 016: ID 1df7:2500 SDRplay RSP1 is s china clone and not supported by sdrplay api
that is why I ask for MDI2500 support like:
https://github.com/f4exb/libmirisdr-4

**Reply by KubaPro010** [reply]
it is supported, thats why it shows up as a RSP1 which is genuine, i have a "RSP1C" which is also a clone, and yet it works with the official API, atleast try

**Reply by dl5raz** [reply]
You are right! did install SDRplay_RSP_API-Linux-3.15.2.run
and cmake option:
-DOPT_BUILD_SDRPLAY_SOURCE=ON \
and it workes!
